src/list.c

```diff
@@ -26,6 +26,7 @@ list_new(void) {
 
 /*
  * Free the list.
+   @self: Pointer to the list 
```

Comment:
Please keep comment style consistent.

```suggestion
 * @self: Pointer to the list 
```

The rest of the comments should be updated to match the above.

Reply:
Finish Refinement, And I also commited it.